.openshiftio/application.yaml

```diff
@@ -125,7 +120,7 @@ objects:
                 periodSeconds: 3
                 successThreshold: 1
                 timeoutSeconds: 1
-              image: nodejs-health-check
+              image: ""
```

Comment:
what is the reason for removing this?

Reply:
Certainly the changes in the container Image property we made because its triggering the two deployments for the single build.
e.g. in the following screenshot once we apply the DC change its triggering two deployments 6-7.

![image](https://user-images.githubusercontent.com/28615674/45994870-3bd08380-c0b3-11e8-9634-d2ac235ef03b.png)